layers/README.md

```diff
@@ -0,0 +1,85 @@
+# layers
+
+This folder contains modules and scripts for working with ATT&CK layers. The core module allows users to load, validate, process, and save ATT&CK layers. A brief overview of the components can be found below. All scripts adhere to the MITRE ATT&CK Navigator Layer file format, [version 3.0](https://github.com/mitre-attack/attack-navigator/blob/develop/layers/LAYERFORMATv3.md).
+
+| script | description |
+|:-------|:------------|
+| [exceptions.py](core/exceptions.py) | Implements errors and supporting functions for the core module. |
+| [filter.py](core/filter.py) | Implements a basic filter object. |
+| [gradient.py](core/gradient.py) | Implements a basic gradient object. |
+| [layer.py](core/layer.py) | Provides an interface for interacting with core module's layer representation. A further breakdown can be found in the corresponding section below. |
+| [layerobj.py](core/layerobj.py) | Implements a basic layer object. This should not be manipulated directly, instead, use the [layer representation](core/layer.py). |
+| [layout.py](core/layout.py) | Implements a basic layout object. |
+| [legenditem.py](core/legenditem.py) | Implements a basic legenditem object. |
+| [metadata.py](core/metadata.py) | Implements a basic metadata object. |
+| [technique.py](core/technique.py) | Implements a basic technique object. |
+| [layerops.py](manipulators/layerops.py) | Provides a means by which to combine multiple ATT&CK layer objects in customized ways. A further breakdown can be found in the corresponding section below. |
+
+## layer.py
```

Comment:
I'll pull them out. The reason I put them in in the first place is due to wanting to be more explicit, since although the average user may want to import things via module, someone is inevitably want to have more direct control over the system, and I wanted to provide some details for those people.